Core/Translation/en_EN.json

```diff
@@ -600,6 +603,7 @@
     "api": "API",
     "enable-api": "Enable API",
     "default-tax": "Default TAX",
+    "renumber-accounting": "Renumber Accounting",
```

Comment:
Codacy is warning you that this field called 'renumber-accounting' is duplicated.